01-Setup and getting started/README.md

```diff
@@ -3,6 +3,8 @@
 
 </br>
 
+> We have updated our auto update feature to use AAD Authentication for Function App deployment instead of basic auth or publishing profile. Please refer details [here](../FAQs/README.md/#6-are-there-any-recommended-settingschanges-for-existing-azts-setup). 
```

Comment:
**As a part of rolling out security improvements in AzTS setup,** we have updated our auto update feature to use AAD Authentication for Function App deployment instead of basic auth or publishing profile.

The "Why" is more important for all such notices.